config/webpack.client.js

```diff
@@ -101,7 +103,13 @@ module.exports = (env, argv) => {
       })
     ]),
     resolve: {
-      modules: [root, 'node_modules']
+      modules: [root, 'node_modules'],
+      alias: {
+        lodash: 'node_modules/lodash-es',
```

Comment:
Does this ensure that every time someone imports something from `lodash` the request gets routed to `loadash-es`? That's very clever 💯

Reply:
Yes! And then only the stuff we use is added to the bundle!